### ISSUE TITLE
dotnet-install should download builds using the aka.ms links

### ISSUE THREAD

**Issue by adiaaida** [dotnet/arcade#6851]
Currently, dotnet-install.ps1/sh uses the latest.version/productVersion.txt file to download builds. Once the aka.ms link generation infrastructure is in place, the dotnet-install scripts should use the aka.ms links to download builds.

**Comment by adiaaida** [comment]
This appears to be part of https://github.com/dotnet/install-scripts/pull/113. Closing in favor of the issue in install-scripts (https://github.com/dotnet/install-scripts/issues/12).
